dicodile/_dicodile.py

```diff
@@ -190,7 +190,6 @@ def dicodile(X, D_init, reg=.1, n_iter=100, eps=1e-5, window=False,
             if verbose > 1:
                 print('[INFO:{}] Resampled atom {}'.format(name, k0))
 
-        # Update the dictionary D_hat in the encoder
```

Comment:
Why remove this comment?

Reply:
That was a mistake I guess, I'll put it back